src/components/preview/preview.style.ts

```diff
@@ -1,14 +1,106 @@
-import styled from "styled-components";
+import styled, { css, keyframes } from "styled-components";
 import { margin } from "styled-system";
 import baseTheme from "../../style/themes/base";
+import { Shapes } from "./preview.component";
 
 const StyledPreview = styled.div`
   ${margin}
 `;
 
+interface StyledPreviewPlaceholderProps {
+  height?: string;
+  width?: string;
+  shape: Shapes;
+  isStatic?: boolean;
+  isLastLine: boolean;
+}
+
+const shimmer = keyframes`
+  0% { 
+    opacity: 0.1 
+  }
+  70% { 
+    opacity: 1
+  }
+  100% { 
+    opacity: 0.1 
+  }
+`;
+
+function getBorderRadius(shape: Shapes) {
+  switch (shape) {
+    case "rectangle-round":
+      return "--borderRadius400";
+    case "circle":
+      return "--borderRadiusCircle";
+    default:
+      return "--borderRadius100";
+  }
+}
+
+function getHeight(shape: Shapes) {
+  switch (shape) {
+    case "rectangle":
+      return "40px";
+    case "rectangle-round":
+      return "40px";
+    case "circle":
+      return "56px";
+    default:
+      return "14px";
+  }
+}
+
+function getWidth(shape: Shapes) {
```

Comment:
suggestion (non-blocking): as there's essentially only 2 values being returned here you could use an if/else to remove one check here but it's not critical etc

```
function getWidth(shape: Shapes) {
  if (shape.includes("rectangle")) {
    return "120px";
  }

  return "100%";
}